doc/pages/blurbs.json

```diff
@@ -504,11 +505,20 @@
   },
   {
     "name": "Jeremiah Tabb",
-    "image": "/assets/contributers/jeremiah_tabb.jpg",
+    "image": "/assets/contributors/jeremiah_tabb.jpg",
```

Comment:
Good catch!

Reply:
For the Julian Macmang image, let's actually keep the original file extension and update the entry in this file to be `jpg` instead of `png`. I believe I already updated this in master. Try rebasing?

Reply:
done.